static/app/components/modals/sudoModal.tsx

```diff
@@ -113,6 +114,12 @@ class SudoModal extends React.Component<Props, State> {
     }
   };
 
+  handleLogout = async () => {
+    const {api} = this.props;
+    await logout(api);
```

Comment:
@maxiuyuan can you wrap this in a try catch? So that we can still call `window.location.assign()` in case `await logout(api)` errors.